### ISSUE TITLE
Nav bar button aria-label is too long

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-pageLevelProgress#170]
### Subject of the issue
The `.nav__pagelevelprogress-btn` `aria-label` is too long. As a general rule, alt text or aria should be as succinct as possible, without sacrificing accuracy.

Current nav bar button aria-label: 
"_Page progress. Use this to listen to the list of regions in this topic and whether they're completed. You can jump directly to any that are incomplete or which sound particularly interesting._ {{percentageComplete}}%"

<img width="926" alt="nav_btn_aria" src="https://user-images.githubusercontent.com/7045330/226678714-a6603376-0b71-4978-8253-514158f8010e.png">

Current PLP drawer aria-label (this is read when the PLP drawer is opened before you interact with the PLP items):
"_Page sections_" 

<img width="953" alt="drawer_list_aria" src="https://user-images.githubusercontent.com/7045330/226678781-1e9e1dda-a835-42ad-9a98-d880ee9a3e61.png">

### Expected behaviour
I think the `.nav__pagelevelprogress-btn` should have a shorter label and additional instruction to be added to the drawer instruction instead.

Suggestion example:

"pageLevelProgressIndicatorBar": "Page progress. {{percentageComplete}}%. Open page sections."
"pageLevelProgress": "List of page sections and completion status. Select incomplete sections to jump directly to the content."
